docs/api/examples/requests-examples.py

```diff
@@ -0,0 +1,52 @@
+from urllib.parse import urlencode
+
+import requests
+
+# Example 1
+# Note the question mark (`?`) marking the start of a query string.
+url = "https://oqt.ohsome.org/api/indicator?"
+parameters = {
+    "name": "GhsPopComparisonBuildings",
+    "layerName": "building_count",
+    "dataset": "regions",
+    "featureId": "3",
+}
+# Use `urllib.parse.urlencode` to ensure proper URL encoding:
+response = requests.get(url + urlencode(parameters))
+assert response.status_code == 200
+
+# Example 2
+# Note the *missing* question mark (`?`).
+url = "https://oqt.ohsome.org/api/indicator"
+parameters = {
+    "name": "GhsPopComparisonBuildings",
+    "layerName": "building_count",
+    "dataset": "regions",
+    "featureId": "3",
+}
+response = requests.post(url, json=parameters)
+assert response.status_code == 200
+
+# Example 3
+# Note the using of the `json` library to dump the GeoJSON as string to the parameters.
```

Comment:
That statement is wrong and should be removed.

Reply:
Yes. That was true before v.0.7.0